data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1878 @@ mission "Stone of our Fathers 5"
+			choice
+				`	"How do I find him?"`
+			`	The Union employee begins moving away from you. "Name's Hroar, you can usually find him in the Red Rock Saloon, inside the Muspel Spaceport."`
+			`	The disgruntled Union employee then says, quite loudly, "No ser! I do not know where the bracelet shop is, please stop asking me!" And hurries away from you.`
+				accept
+	on complete
+		dialog
+			`	You've arrived at the rusty hued world that is Muspel. You should head into the spaceport to see if you can find Hroar.`
+
+
+
+mission "Home for Skadenga 9"
+	name "Home for Skadenga"
+	minor
+	invisible
+	to offer
+		has "Home for Skadenga 8: done"
+	source "Muspel"
+	destination
+		government "Republic" "Neutral" "Syndicate" "Free Worlds" "Deep"
+	on offer
+		event "hroar message" 13
+		conversation
+			`	The floor of the spaceport is covered in the dirty red dust of hundreds of shuffling feet. Prospectors, miners, drillers, and all their attendant associates are intermixed here, gossiping about the latest finds, accidents, and hauls. Every person either saturated in red earth, or immaculately clean, creating a visual aid to boundaries of class, paycheck and leadership.`
+			`	The Red Rock Saloon is located deep in the trade-craft section of the spaceport, far enough from the tourist areas to discourage anyone, but the locals, and those with business to enter.`
+			scene "scene/redrocks"
+			choice
+				`	(Enter the Red Rock Saloon.)`
+			`	The doors swing open revealing a dimly lit room filled with rough unsavory people. Piano music is playing on a cheap speaker, and the eyes' of the patrons slide to get a good look at you without turning their heads.`
+			`	There is an expectant pause. You get the feeling they are waiting for you to either announce something or leave.`
+			choice
+				`	"I am looking for Hroar."`
+			`	For a moment nothing happens and then a gruff voice spirals out from a dark corner. "Back here."`
+			`	You walk to the back of the saloon and find a older man with long grey handlebar mustache seated with his back against the wall, and a bottle of whiskey on his otherwise empty table.`
+			scene "scene/hroar"
+			`	"Have a seat", He says.`
+			`	Once seated, he places a gun on the table.`
+			`	"So the Allen boys think they can send some green-horned killer to take me out?" Hroar barks a laugh. "Not likely going to happen."`
+			`	With his free hand he takes a swig of his whiskey. "Here's what's going to happen. I am not going to kill you. Only maim you, a bit. And you tell the Allen boys that the next kid they send will come back even worse. We had a deal and I expect them to keep it."`
+			choice
+				`	"You got the wrong person. I don't even know who the Allen boys are.`
+					goto next
+				`	"I was not sent by the Allen whoevers, and if you even touch me, I will burn this entire planet to ash."`
+					goto buffcheck
+			label buffcheck
```

Comment:
```suggestion
```

Unless you have somewhere else that also sends the conversation to label buffcheck, this goto/label pair isn't needed. Whenever you have a choice, an option that doesn't specify a "goto" will always just go to the next line, just like the various times you've used it in the conversations previously.


It works both ways, too. For example, in the following block, choosing "you got the wrong person" will bypass the stuff listed under other choices to continue outside the choice with "we're all friends having a drink." While picking the gunfight option will send it to the combat label. (please excusing the indentation. GitHub text boxes don't handle tabs nicely)
```
		choice
			`	"You got the wrong person.`
			`	"Gunfight"`
				goto combat
			`We're all friends having a drink.`
				decline
			label combat
			`Time to have a shoot-out.`

```